.circleci/dynamic_config.yml

```diff
@@ -72,10 +74,20 @@ executors:
     working_directory: ~/ng
     resource_class: large
 
+  bazel-executor:
+    parameters:
+      nodeversion:
+        type: string
+        default: *default_nodeversion
+    docker:
+      - image: cimg/node:<< parameters.nodeversion >>
+    working_directory: ~/ng
+    resource_class: 2xlarge
+
   windows-executor:
     # Same as https://circleci.com/orbs/registry/orb/circleci/windows, but named.
     working_directory: ~/ng
-    resource_class: windows.medium
+    resource_class: windows.2xlarge
```

Comment:
This will increase the credit usage drastically  500 credits/min compared previously 40 credits/min

https://circleci.com/product/features/resource-classes/#dev-windows

Reply:
//cc @josephperrott.

Reply:
I'll try decreasing it again. This was partially to understand why things were flaky or failing which I think was from running 8+ tests concurrently on a single machine. Now it's running 6 in bazel spread over 4 CI shards (the `parallelism` param).